source/dub/internal/vibecompat/inet/path.d

```diff
@@ -283,7 +283,7 @@ struct PathEntry {
 
 	string toString() const { return m_name; }
 
-	Path opBinary(string OP)(PathEntry rhs) const if( OP == "~" ) { return Path(cast(immutable)[this, rhs], false); }
+	Path opBinary(string OP)(PathEntry rhs) const if( OP == "~" ) { return Path(([this, rhs]).idup, false); }
```

Comment:
Okay, thanks! I'll just apply to workaround at the top then. Good luck with your move!

(The immutable stuff is safe, because all input is `immutable`, too. I'll update master to just avoid the casts.)